ntbs-service-unit-tests/Jobs/DataQualityAlertsJobTest.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Hangfire;
+using Moq;
+using ntbs_service.DataAccess;
+using ntbs_service.Jobs;
+using ntbs_service.Models;
+using ntbs_service.Models.Entities;
+using ntbs_service.Services;
+using Xunit;
+
+namespace ntbs_service_unit_tests.Jobs
+{
+    public class DataQualityAlertsJobTest
+    {
+        private readonly Mock<IAlertService> _mockAlertService;
+        private readonly Mock<IDataQualityRepository> _mockDataQualityRepository;
+
+        private readonly DataQualityAlertsJob _dataQualityAlertsJob;
+
+        public DataQualityAlertsJobTest()
+        {
+            _mockAlertService = new Mock<IAlertService>();
+            _mockDataQualityRepository = new Mock<IDataQualityRepository>();
+            
+            _dataQualityAlertsJob = new DataQualityAlertsJob(
+                _mockAlertService.Object,
+                _mockDataQualityRepository.Object);
+        }
+
+        [Fact]
+        public async Task OnRun_WithEligibleNotifications_CallsExpectedMethodWithEligibleNotifications()
```

Comment:
I would probably be inclined to test that if there are no eligible notifications that it doesn't call the alert service methods. 

If I could come up with a clean way to test that multiple notifications are called then I might be inclined there too... but I can't come up with something particularly clean without digging into it further.

Reply:
Sounds good, I originally tried the latter and it wasn't clean/didn't quite work sadly, happy to look into again if we think it's worth the time though